<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_state">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_picture">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_showImage">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getImage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="horizontalRotation">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation Horizontal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="verticalRotation">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation Verticval</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="exposure">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Night-Mode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="resolution">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>High resolution</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="infraredOn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn On Infrared</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="restartServer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart the Server</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="closeConnection">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Connetion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
